diff --git a/node_modules/react-native/index.js b/node_modules/react-native/index.js
index f556fa3..79f18c7 100644
--- a/node_modules/react-native/index.js
+++ b/node_modules/react-native/index.js
@@ -449,6 +449,7 @@ module.exports = {
         "'deprecated-react-native-prop-types' package.",
     );
     return require('deprecated-react-native-prop-types').ViewPropTypes;
+    
   },
 };
 
diff --git a/node_modules/react-native/scripts/.packager.bat b/node_modules/react-native/scripts/.packager.bat
new file mode 100644
index 0000000..e89260e
--- /dev/null
+++ b/node_modules/react-native/scripts/.packager.bat
@@ -0,0 +1 @@
+set RCT_METRO_PORT=8081
\ No newline at end of file
